hardware/computemodule/cm-peri-sw-guide.md

```diff
@@ -158,8 +158,6 @@ For each of the examples we assume a CM+CMIO or CM3+CMIO3 board with a clean ins
 
 The examples here require internet connectivity, so a USB hub plus keyboard plus WiFi or Ethernet dongle plugged into the CMIO USB port is recommended.
 
-If you suspect any issues or bugs with Device Tree it is always best to try a `sudo rpi-update` to make sure you are using the latest firmware. 
-
 WARNING: if you have edited any of the default .dtb files in `/boot` or `/boot/overlays` these may be overwritten by rpi-update.
```

Comment:
If you're removing the line above, then this following line should be removed too :wink: